gallery/src/components/NxTreeView/NxTreeViewPage.tsx

```diff
@@ -93,26 +93,31 @@ const NxTreeViewPage = () =>
     </GalleryDescriptionTile>
 
     <GalleryTile title="NxTreeView Basic Example">
+      <h4 className="nx-h4">Example:</h4>
```

Comment:
These changes overlap and conflict with the changes I'm making in RSC-125.

Reply:
> These changes overlap and conflict with the changes I'm making in RSC-125.

https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/c1bee24e1bad88cdd3df1a1951277a3709d8e8e0